### ISSUE TITLE
problem on how to run the STAR (command not found)

### ISSUE THREAD

**Issue by karosse** [alexdobin/STAR#331]
Hello,
I already done 1. cd source 2. make STARforMacStatic CXX=/path/to/gcc 3.make STARforMacStatic
I tried to generate genome indexes files 
1. cd source 2.STAR --runThreadN 3--runMode genomeGenerate--genomeDir /Users/zty/Desktop/Dir--genomeFastaFiles /Users/zty/Desktop/gene/gencode.vM15.transcripts.fa--sjdbGTFfile /Users/zty/Desktop/gene/gencode.vM15.annotation.gtf--sjdbOverhang 284
but I got : -bash: STAR: command not found
also I 1. cd /STAR-master/bin/MacOSX_x86_64 2.STAR--opt
but still -bash: STAR: command not found

**Comment by alexdobin** [comment]
Hi @karosse 

with bash, you need to specify the path to the executable explicitly, or add it to the path variable. If you are running from within a directory that contains STAR executable (e.g. STAR compilation directory), you need:
$ ./STAR ...
Note that the Log.out file will be produced in that directory. I would recommend running STAR from the working directory (in your case /Users/zty/Desktop/Dir/), but specifying full pass to the executable, i.e.:
$ cd /Users/zty/Desktop/Dir/
$ /path/to/STAR/source/STAR ...

Cheers
Alex

**Comment by shwetagodbole** [comment]
Hi , I am facing the same issue. I am using my university's cluster too. Installed star > source STAR > make STAR 
make star up to date. 
When i run STAR it says command not found. 
I have also added the correct path to nano ~/.bashrc 
Where the path is : 
export PATH="/home/godbole/RNA/STAR-2.7.6a":$PATH
I did source ~/.bashrc
closed and reopened the terminal and it does not find the command still. 
I had installed it a few days ago and did all the steps above and it worked fine. After 2 days when I wanted to actually start with the analysis it does not work anymore. Could the file be corrupted? 
Any help will be great! Thanks again
